### PR TITLE
Tooltips and FAQ Rewording

### PR DESCRIPTION
**Pull request by domdreams**
Issues #97 , #98

### CODE REVIEW

File: src/app/components/candidate-card-expanded/candidate-card-expanded.component.html [hunk 1]
```diff
@@ -69,12 +69,12 @@ <h2>{{ candidate['description'] }}</h2>
...
       <div class="title">
         <p>
-          Donors by Group
+          Donations by Group
           <sup>
             <i
               class="helper-tip fa fa-question-circle"
               aria-hidden="true"
-              matTooltip="Description goes here."
+              matTooltip="Total contributions from five largest groups of campaign donors.  Groups are determined by the industry segment of each donors’ employer."
```
Comment (nancyheiss) [inline src/app/components/candidate-card-expanded/candidate-card-expanded.component.html:77]: I think it sounds better "... the five largest groups...".

**Review by nancyheiss — CHANGES_REQUESTED** [review]

File: src/app/components/candidate-card-expanded/candidate-card-expanded.component.html (hunk 1 above)
Comment (domdreams) [reply src/app/components/candidate-card-expanded/candidate-card-expanded.component.html]: @nancyheiss Done

File: src/app/components/candidate-card-expanded/candidate-card-expanded.component.html [hunk 2]
```diff
@@ -69,12 +69,12 @@ <h2>{{ candidate['description'] }}</h2>
...
       <div class="title">
         <p>
-          Donors by Group
+          Donations by Group
           <sup>
             <i
               class="helper-tip fa fa-question-circle"
               aria-hidden="true"
-              matTooltip="Description goes here."
+              matTooltip="Total contributions from the five largest groups of campaign donors.  Groups are determined by the industry segment of each donors’ employer."
```
Comment (nancyheiss) [inline src/app/components/candidate-card-expanded/candidate-card-expanded.component.html:77]: typo: should be "donor's"

**Review by nancyheiss — CHANGES_REQUESTED** [review]

File: src/app/components/candidate-card-expanded/candidate-card-expanded.component.html (hunk 2 above)
Comment (domdreams) [reply src/app/components/candidate-card-expanded/candidate-card-expanded.component.html:76]: Ok added

**Review by nancyheiss — APPROVED** [review]

**Review by gSoD17 — APPROVED** [review]

**Pull request merged**